stdlib/stdlib.c

```diff
@@ -31,10 +31,10 @@ void _$populate_dynamic_array(uint64_t* address, uint64_t value, int64_t should_
     if (address == 0)
         return;
 
-    uint64_t length = *(address - 1);
+    uint64_t length = *(address + 1);
```

Comment:
It seems to me that this method assumes that `address` parameter is given with different offset that the same parameter in `array_ref_count_decrement`, which is confusing.

Reply:
I'll unify this